app/src/main/res/drawable/ic_summary_call_icon.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+  <item android:drawable="@drawable/summary_call_icon_background" />
+  <item android:drawable="@drawable/summary_call_icon_foreground" />
```

Comment:
We can maybe use the insets inside the item instead of creating a new drawable, WDYT?
(for background drawable as well)

```suggestion
  <item>
    <inset
      android:drawable="@drawable/ic_call_9dp"
      android:inset="@dimen/spacing_4" />
  </item>
```

Reply:
I completely forgot that I could inline these. Done!